README.md

```diff
@@ -1,35 +1,32 @@
-5.  `cd ../build/llvm-project`
-6.  Run cmake command to create build files (make or ninja) with default build type (Debug).
-
-     Support for X86-64 and ARM raisers will be built into the tool based on the LLVM build targets. There is no interdependency. Consequently,  support to raise only X86-64 binaries is built during X86-only LLVM builds; support to raise only ARM binaries is built during ARM-only LLVM builds. The tool is not built during an LLVM build with targets that do not include either X86 or ARM.
-
-    For e.g., either of the following `cmake` commands is known to build the tool and its dependencies.
-
-    `cmake -G "Ninja" -DCMAKE_INSTALL_PREFIX=</full/path/to/github/install/llvm-project> </full/path/to/github/src/llvm-project/llvm> -DLLVM_ENABLE_PROJECTS=clang`
-
-    `cmake -G "Ninja" -DCMAKE_INSTALL_PREFIX=</full/path/to/github/install/llvm-project> </full/path/to/github/src/llvm-project/llvm> -DLLVM_TARGETS_TO_BUILD="`*TARGET_ARCH*`" -DLLVM_ENABLE_PROJECTS=clang` 
-
-     The corresponding `cmake` commands known to work for Release builds are as follows:
+1.  Setup the source tree:
+```sh
+git clone --depth 100 -b master https://github.com/llvm/llvm-project.git
+cd llvm-project && git clone -b master https://github.com/microsoft/llvm-mctoll.git llvm/tools/llvm-mctoll
+```
+2.  Build [LLVM with CMake](https://llvm.org/docs/CMake.html#frequently-used-cmake-variables)
 
-     `cmake -G "Ninja" -DCMAKE_INSTALL_PREFIX=</full/path/to/github/install/llvm-project> </full/path/to/github/src/llvm-project/llvm> -DLLVM_TARGETS_TO_BUILD="`*TARGET_ARCH*`" -DCMAKE_BUILD_TYPE=Release -DLLVM_ENABLE_DUMP=ON -DLLVM_ENABLE_ASSERTIONS=ON -DLLVM_ENABLE_PROJECTS=clang`
+    Support for X86-64 and ARM raisers will be built into the tool based on the LLVM build targets. There is no interdependency. Consequently,  support to raise only X86-64 binaries is built during X86-only LLVM builds; support to raise only ARM binaries is built during ARM-only LLVM builds. The tool is not built during an LLVM build with targets that do not include either X86 or ARM.
 
-    `cmake -G "Ninja" -DCMAKE_INSTALL_PREFIX=</full/path/to/github/install/llvm-project> </full/path/to/github/src/llvm-project/llvm> -DCMAKE_BUILD_TYPE=Release -DLLVM_ENABLE_DUMP=ON -DLLVM_ENABLE_ASSERTIONS=ON -DLLVM_ENABLE_PROJECTS=clang`
+    For example:
+```sh
+mkdir build && cd build
+cmake -G "Ninja" -DLLVM_TARGETS_TO_BUILD=X86;ARM -DLLVM_ENABLE_PROJECTS=clang -DLLVM_ENABLE_DUMP=ON -DLLVM_ENABLE_ASSERTIONS=ON -DCMAKE_BUILD_TYPE=Release ..
+ninja llvm-mctoll
+```
 
-    You may also use the option `-G "Unix Makefiles"` instead of `-G "Ninja"`.
+## Windows notes
 
-7.  Run `make llvm-mctoll` or `ninja llvm-mctoll`
+Build using the `x64 Native Tools Command Prompt` with Ninja as above or using the Visual Studio generator for development.
+Also see https://llvm.org/docs/GettingStartedVS.html
 
```

Comment:
I think the `Windows Notes` section should spell out that the tool does not yet raise Windows (PE) binaries.